packages/react/src/themes/teams/components/Table/tableCellStyles.ts

```diff
@@ -0,0 +1,44 @@
+import { ComponentStyleFunctionParam, ICSSInJSStyle } from '../../../types'
+import { TeamsTableVariables } from './tableVariables'
+import { TableCellProps } from '../../../../components/Table/TableCell'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+
+export default {
+  root: ({
+    variables: v,
+    theme: { siteVariables },
+  }: ComponentStyleFunctionParam<TableCellProps, TeamsTableVariables>): ICSSInJSStyle => {
+    const borderFocusStyles = getBorderFocusStyles({
+      siteVariables,
+    })
+
+    return {
+      display: 'flex',
+      flexFlow: 'row nowrap',
+      flexGrow: 1,
+      flexBasis: 0,
+      minWidth: v.minCellWidth,
+      outline: 0,
+      borderWidth: v.borderWidth,
+      borderStyle: 'solid',
+      borderColor: 'transparent',
+      ':focus': borderFocusStyles[':focus'],
```

Comment:
can we just spread `borderFocusStyles` instead of separatelly spreading `":focus"` and `":focus-visible"`?

Reply:
Yep. will update that :)
Thank you!